message.py

```diff
@@ -17,16 +17,17 @@
 import smtplib
 from email.mime.text import MIMEText
 from moc_utils import get_absolute_path
+from exception import BadEmailRecipient
```

Comment:
from exception**s**

This suggests that you have not tested this, you should be testing that you can raise BadEmailRecipient here (how to do this: comment out everything inside the  'if __name__ == '__main__' and add a line to raise the exception there, run the script to see if it works, - once it does, remove the additional test line and uncomment the real code.)